src/main/java/uk/gov/justice/digital/hmpps/prisonstaffhub/services/HealthInfo.java

```diff
@@ -21,10 +18,8 @@ public Health health() {
         return Health.up().withDetail("version", getVersion()).build();
     }
 
```

Comment:
Not keen on losing these javadocs - data served here is wide open and public

Reply:
I just copied keyworker-api

Reply:
I just went with whatever keyworker-api does. Not sure why there different.